### ISSUE TITLE
Gamemaker Studio 2 games are not supported

### ISSUE THREAD

**Issue by I-CyberDaun-I** [puggsoy/GMS-Explorer#4]
Nothing happens. When I try to open the win file from Wicked Games, the program gives the following error, which can be seen in the screenshot. 
https://imgur.com/G46TVQN

**Comment by puggsoy** [comment]
I've seen this happen with games made in Game Maker Studio 2, and GMS Explorer only supports those made in GMS 1. I'll have to look into fixing this since it's a pretty major issue. Can't really say when that will happen unfortunately.

**Comment by I-CyberDaun-I** [comment]
Okay, I'll wait for an update.

чт, 6 июл. 2023 г. в 08:43, Sami Habib ***@***.***>:

**Comment by Windrainfox** [comment]
Wondering if there is a workable solution now?

**Comment by I-CyberDaun-I** [comment]
i dont know

сб, 23 сент. 2023 г. в 12:10, Windrainfox ***@***.***>:

**Comment by luxidoptera** [comment]
Any updates?

**Comment by puggsoy** [comment]
Nope, sorry. This project is very low on my current priority list.

**Comment by Hexaae** [comment]
Same here trying to open Blazing Chrome data.win (I wanted to fix annoying scrolling hitches, and fix a bug)...

![image](https://github.com/user-attachments/assets/89e7e54a-9877-4c09-920e-c9ba97ef7b6c)

**Comment by Tenqui-Noir** [comment]
I also encountered this problem....
